cmd/release-controller/types.go

```diff
@@ -251,6 +277,8 @@ const (
 
 	releaseAnnotationFromTag = "release.openshift.io/from-tag"
 	releaseAnnotationToTag   = "release.openshift.io/tag"
+
+	releaseAnnotationFromRelease = "release.openshift.io/from-release"
```

Comment:
document the format you expect the value of this annotation to have (what is the annotation value in human terms)